src/Sql/Sql.Sdk/Generated/Models/ExtendedServerBlobAuditingPolicy.cs

```diff
@@ -116,7 +116,7 @@ public ExtendedServerBlobAuditingPolicy()
         /// audit logs.
         ///
         /// For more information, see [Database-Level Audit Action
-        /// Groups](https://learn.microsoft.com/en-us/sql/relational-databases/security/auditing/sql-server-audit-action-groups-and-actions#database-level-audit-action-groups).
```

Comment:
Shouldn't we keep `learn.microsoft` links?

Reply:
Autorest.csharp has not updated this static string I believe as we are using an old version for track1.

Reply:
Reverted the change